estela-web/src/shared/notificationsSidenav/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import { Menu, Layout } from "antd";
+
+import "./styles.scss";
+
+const { Sider } = Layout;
+
+interface NotificationsInboxPropsInterface {
+    path: string;
+}
+
+export class NotificationsSidenav extends Component<NotificationsInboxPropsInterface, unknown> {
+    path = this.props.path;
+    render(): JSX.Element {
+        return (
+            <Sider width={240}>
+                <Menu mode="inline" className="h-full" selectedKeys={[`${this.path}`]} defaultOpenKeys={["2", "5"]}>
+                    <div>
+                        <p className="m-5 text-estela-black-medium text-base">NOTIFICATIONS</p>
+                    </div>
+                    <Menu.Item key={"/notifications/inbox"} className="">
+                        <div className="flex items-center stroke-black hover:stroke-estela hover:bg-button-hover hover:text-estela rounded">
+                            <Link to={`/notifications/inbox`}>Inbox</Link>
+                        </div>
+                    </Menu.Item>
+                    <Menu.Item key={"/notifications/settings"} className="">
+                        <div className="flex items-center stroke-black hover:stroke-estela hover:bg-button-hover hover:text-estela rounded">
+                            <Link to={`/notifications/settings`}>Settings</Link>
+                        </div>
```

Comment:
Set notification margin left. It's to close to the sidebar
<img width="245" alt="Screenshot 2022-10-27 at 10 05 49" src="https://user-images.githubusercontent.com/33165861/198327176-18d0bffc-588f-4794-8b3c-2d3a4c4543ce.png">